src/main/java/jenkins/plugins/logstash/kibana/EmbeddedLogActionFactory.java

```diff
@@ -28,8 +29,8 @@
 
     @Override
     public Collection<? extends Action> createFor(Run target) {
-        return Arrays.<Action>asList(new KibanaEmbeddedLogAction(target), 
-                new ElasticsearchIncrementalLogAction(target));
+        return target instanceof WorkflowRun ? Collections.singleton(new KibanaEmbeddedLogAction(target)) :
```

Comment:
Yep, it was just a part of the PoC. I'm not sure if it worth merging both log actions